src/main/java/de/tum/cit/aet/artemis/assessment/repository/cleanup/FeedbackCleanupRepository.java

```diff
@@ -43,6 +43,23 @@ WHERE f.result IN (
             """)
     int deleteFeedbackForOrphanResults();
 
+    /**
+     * Counts {@link Feedback} entries where the associated {@link Result} has no submission and no participation.
+     *
+     * @return the number of entities that would be deleted
+     */
+    @Query("""
+            SELECT COUNT(f)
+            FROM Feedback f
+            WHERE f.result IN (
```

Comment:
this query might be very expensive, in particular the `IN`-comparison when the subquery returns a large results. Is it expected to be rather small as some kind of "edge-case" cleanup?